setup.py

```diff
@@ -28,6 +28,8 @@
     "absl-py",
     "pytest",
     "pyink",
+    "rioxarray",
+    "rasterio",
```

Comment:
We don’t want to incur such a heavy dependency. Further, under the hood, I believe rasterio just makes calls to pyproj.

Reply:
Read a bit more. Sorry for assuming what the PR did!